go/consensus/tendermint/tendermint.go

```diff
@@ -1357,6 +1360,7 @@ func init() {
 	Flags.Uint64(CfgConsensusMinGasPrice, 0, "minimum gas price")
 	Flags.Uint64(CfgConsensusSubmissionGasPrice, 0, "gas price used when submitting consensus transactions")
 	Flags.Uint64(CfgConsensusSubmissionMaxFee, 0, "maximum transaction fee when submitting consensus transactions")
+	Flags.Bool(CfgConsensusDisableCheckTx, false, "do not perform CheckTx on incoming transactions")
```

Comment:
This 100% should be hidden and behind the "this is unsafe" feature gate.

Reply:
Yup, forgot about it. Should be changed now.